stock_valuation_layer_usage/views/stock_move_views.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="view_move_form_inherit" model="ir.ui.view">
+        <field name="name">stock.move.form.inherit</field>
+        <field name="model">stock.move</field>
+        <field name="inherit_id" ref="stock_account.view_move_form_inherit" />
+        <field name="arch" type="xml">
+            <group name="linked_group" position="after">
+                <group
+                    name="layers_used"
+                    string="Valuation Layers"
+                    groups="account.group_account_user"
```

Comment:
Could you add `colspan="2"` to improve visual effect?

![2025-03-21_11-37](https://github.com/user-attachments/assets/8c313884-ea66-44ef-9799-4157fdb04913)

Reply:
Updated, Thanks